newrelic/resource_newrelic_workflow.go

```diff
@@ -0,0 +1,371 @@
+package newrelic
+
+import (
+	"context"
+	"fmt"
+	"log"
+	"strings"
+
+	"github.com/newrelic/newrelic-client-go/pkg/workflows"
+
+	"github.com/hashicorp/terraform-plugin-sdk/v2/diag"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/validation"
+	"github.com/newrelic/newrelic-client-go/pkg/ai"
+	"github.com/newrelic/newrelic-client-go/pkg/errors"
+)
+
+func resourceNewRelicWorkflow() *schema.Resource {
+	return &schema.Resource{
+		CreateContext: resourceNewRelicWorkflowCreate,
+		ReadContext:   resourceNewRelicWorkflowRead,
+		UpdateContext: resourceNewRelicWorkflowUpdate,
+		DeleteContext: resourceNewRelicWorkflowDelete,
+		Importer: &schema.ResourceImporter{
+			State: schema.ImportStatePassthrough,
+		},
+		Schema: map[string]*schema.Schema{
+			// Required
+			"name": {
+				Type:        schema.TypeString,
+				Required:    true,
+				Description: "(Required) The name of the workflow.",
+			},
+			"destination_configurations": {
+				Type:        schema.TypeList,
```

Comment:
the order of these shouldn't matter, should it?  Why is this not `schema.TypeSet`?

Reply:
Correct, we can make this a `TypeSet`